lib/trln_argon/argon_search_builder/share_bookmarks.rb

```diff
@@ -0,0 +1,16 @@
+module TrlnArgon
+  module ArgonSearchBuilder
+    module ShareBookmarks
+      def get_shared_bookmark_ids(solr_parameters)
```

Comment:
This method name is misleading. I'd prefer something like: `add_document_ids_query`